code/modules/modular_computers/computers/item/computer_ui.dm

```diff
@@ -78,6 +78,9 @@
 		return data
 
 	data["show_imprint"] = istype(src, /obj/item/modular_computer/pda)
+	data["alert_style"] = get_security_level_relevancy()
+	data["alert_color"] = SSsecurity_level?.current_security_level?.announcement_color
+	data["alert_name"] = SSsecurity_level?.current_security_level?.name_shortform
```

Comment:
This shouldn't be on static data, like you show in your gif the alert level doesn't update until you close and reopen the yo